src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -173,14 +175,52 @@ admin_orders_update_invoice_note:
     orderId: \d+
     orderInvoiceId: \d+
 
-admin_search_product_for_order_creation:
```

Comment:
This is the doublon route right? Thanks for removing it 😉

Reply:
Thanks @sarjon